### PR TITLE
JUser bind

### PR DESCRIPTION
**Pull request by fanno**
...w users because the values are not yet set, Fixed by moving the check below setProperties

also see:
https://groups.google.com/forum/?fromgroups=#!topic/joomlabugsquad/ggmaWUzb-lM

This is my first pull request so bare with me

### CODE REVIEW

File: libraries/joomla/user/user.php [hunk 1]
```diff
@@ -615,6 +597,24 @@ public function bind(&$array)
 			return false;
 		}
 
+        // Check that username is not greater than 150 characters
```
Comment (eddieajau) [inline libraries/joomla/user/user.php:600]: Seems to be one too many indents here?

**Comment by eddieajau** [comment]
I don't suppose you're able to do the unit test for the bind method while you are at it?

https://github.com/joomla/joomla-platform/blob/b4c34af90a8ea35a510eebfb49fdf7baa8bf990d/tests/suites/unit/joomla/user/JUserTest.php#L394

File: libraries/joomla/user/user.php (hunk 1 above)
Comment (fanno) [inline libraries/joomla/user/user.php:600]: This is my first pull request so i do not know what the normal process is ? i do not understand your comment/note ??

**Comment by fanno** [comment]
sure i can, i just don't know how, is there a tutorial for it somewhere ?

**Comment by fanno** [comment]
hmm should the check for to long password not throw an error ? now that i have looked a bit more at it? or else how would you be able to check the login if the salt has changed because it have been cut down in length ? (just thinking out laud)

**Comment by fanno** [comment]
like this ? 
https://github.com/joomla/joomla-platform/pull/1743

**Pull request closed without merging**